Sources/Testing/Traits/TimeLimitTrait.swift

```diff
@@ -10,17 +10,16 @@
 
 /// A type that defines a time limit to apply to a test.
 ///
-/// To add this trait to a test, use one of the following functions:
-///
-/// - ``Trait/timeLimit(_:)-4kzjp``
+/// To add this trait to a test, use ``Trait/timeLimit(_:)-4kzjp``.
 @available(_clockAPI, *)
 public struct TimeLimitTrait: TestTrait, SuiteTrait {
   /// A type representing the duration of a time limit applied to a test.
   ///
-  /// This type is intended for use specifically for specifying test timeouts
-  /// with ``TimeLimitTrait``. It is used instead of Swift's built-in `Duration`
-  /// type because test timeouts do not support high-precision, arbitrarily
-  /// short durations. The smallest allowed unit of time is minutes.
+  /// Use this type to specify a test timeout with ``TimeLimitTrait``.
+  /// Use this type instead of Swift's built-in `Duration`
+  /// type because the testing library doesn't support high-precision,
+  /// arbitrarily short durations for test timeouts.
+  /// The smallest allowed unit of time is minutes.
```

Comment:
Purely stylistic, but it seems like we could re-flow the sentences here to condense these lines

Reply:
I re-worded this to avoid saying "Use this type" twice and it ended up just as long, but see what you think.